administrator/language/de-DE/plg_editors_tinymce.ini

```
@@ -97,4 +97,6 @@ PLG_TINY_TOOLBAR_BUTTON_FONTSIZESELECT="Schriftgröße auswählen"
 PLG_TINY_TOOLBAR_BUTTON_FORMATSELECT="Format auswählen"
 PLG_TINY_TOOLBAR_BUTTON_SEPARATOR="Trennzeichen"
 PLG_TINY_TOOLBAR_BUTTON_STYLESELECT="Stilauswahl"
+PLG_TINY_FIELD_SANDBOX_IFRAMES_LABEL="Sandbox iFrames"
+PLG_TINY_FIELD_SANDBOX_IFRAMES_DESC="Dies ist eine Sicherheitsfunktion, die die Möglichkeiten von iFrames einschränkt, indem für jeden iFrame ein Sandbox-Attribut gesetzt wird. Aus Sicherheitsgründen wird empfohlen, diese Funktion zu aktivieren. Nur deaktivieren, wenn der iFrame nicht geladen wird und der Inhalt des iFrames sicher ist."
```

```suggestion
PLG_TINY_FIELD_SANDBOX_IFRAMES_DESC="Dies ist eine Sicherheitsfunktion, die die Möglichkeiten von iFrames einschränkt, indem für jeden iFrame das Sandbox-Attribut gesetzt wird. Aus Sicherheitsgründen wird empfohlen, diese Funktion zu aktivieren. Diese Option sollte nur deaktiviert, werden wenn der iFrame nicht geladen wird und der Inhalt des iFrames sicher ist."
```
plus ggfs link zu einer Erklärung was das Sandbox Attribute bei iframes kann und macht. Also z.B.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox